stdlib/result.gr

```diff
@@ -92,30 +204,30 @@ export let peekErr = (fn, r) => {
  *
  * @param msg: The message to prepend if the result contains an `Err`
  * @param result: The result to extract a value from
- * @returns The value inside an `Ok` result
+ * @returns The unwrapped `ok` value if the result is of the `Ok` variant, otherwise throws `msg: err`
  *
- * @example Result.expect("Unexpected error", Ok(1234))
+ * @example Result.expect("Unexpected error", Ok(1234)) == 1234
```

Comment:
I feel like adding the `== 1234` here makes it feel like the expect is some sort of an assertion. I wonder if we just did something like `+ 42` if it wouldn't read that way.